app/startApp.ts

```diff
@@ -1,8 +1,9 @@
-import { configcontainer } from './Dic';
+import { configcontainer } from './dic';
 import App from './App';
+import { AwilixContainer } from 'awilix';
 
 configcontainer()
-  .then(function(container) {
+  .then(function(container: AwilixContainer) {
```

Comment:
Same thing here that this explicit type shouldn't be needed. It should infer the type to `AwilixContainer `